config/config.go

```diff
@@ -339,6 +339,14 @@ type RPCConfig struct {
 	// global HTTP write timeout, which applies to all connections and endpoints.
 	// See https://github.com/tendermint/tendermint/issues/3435
 	TimeoutBroadcastTxCommit time.Duration `mapstructure:"timeout_broadcast_tx_commit"`
+
+	// The name of cert file that used to serve RPC based on Tls.
+	// NOTE: useful when tls_key_file is present
+	TlsCertFile string `mapstructure:"tls_cert_file"`
```

Comment:
```suggestion
	TLSCertFile string `mapstructure:"tls_cert_file"`
```
let's capitalize TLS